fdbrpc/Locality.cpp

```diff
@@ -44,6 +44,7 @@ ProcessClass::Fitness ProcessClass::machineClassFitness(ClusterRole role) const
 		case ProcessClass::CoordinatorClass:
 		case ProcessClass::TesterClass:
 		case ProcessClass::StorageCacheClass:
+		case ProcessClass::BlobWorkerClass:
```

Comment:
Not technically needed in this case (and for TLog) because the default is `NeverAssign`. But, if the default ever changes, we still want to assign the fitness as `NeverAssign` for the `BlobWorkerClass` (which is also likely why we spell it out for others like `StorageCacheClass`).